docs/decisions/api-browser-security.md

```diff
@@ -0,0 +1,74 @@
+# API Security when calling APIS from a Browser Application
+
+## Context and Problem Statement
+
+Currently re-users of the Open Food Facts APIs have two options for authentication, where this is required:
+
+- pass the username and password with each request using HTTP Basic Authentication
```

Comment:
as query or form parameters in fact, not HTTP basic auth (with the Authorization http header)